testkit-core/src/main/scala/org/bitcoins/testkitcore/dlc/DLCTest.scala

```diff
@@ -880,8 +880,11 @@ trait DLCTest {
         .get
         ._2
 
+      val neededPadding =
+        singleOracleInfo.announcement.eventTLV.eventDescriptor.noncesNeeded - digitsToSign.digits.length
+      val paddedDigits = digitsToSign.digits ++ Vector.fill(neededPadding)(0)
```

Comment:
`padRight`

Reply:
So this doesn't seem to be available on a `Vector`.  I tried using `.padTo` but it didn't seem to work.